guides/source/security.md

```diff
@@ -1206,6 +1206,15 @@ TIP: Learn more about credentials with `bin/rails credentials:help`.
 
 WARNING: Keep your master key safe. Do not commit your master key.
 
+#### Accessing environment specific credentials file
+
+In order to access environment specific credential file, run `bin/rails credentials:edit --environement=environment_name`
+
+For example, to edit a production credential file, you need to run
```

Comment:
```suggestion
For example, to edit a production credential file, you need to run:

```

Reply:
@navdevl Could you also add the newline between the block? :bow:

Reply:
@zzak thanks for the review comments. I have made the changes now. 😄

Reply:
@zzak Just asking out of curiosity. I am pushing on the same commit by squashing it. Is that how it should be? Or I should be adding a new commit for every review happening and squash them at the last? How things are done usually? 😅😅 _(I know the question sounds silly)_

Reply:
@Navdevl Squashing to a single commit is preferred, so what you're doing is exactly perfect :)

But don't worry about doing it everytime, sometimes it's better to wait for a committer or core to review before squashing since they may have more changes.

Reply:
Oh, that makes sense. Understood. Thanks @zzak! :))